### PR TITLE
mep12 changes to axes_props.py

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

File: examples/pylab_examples/axes_props.py [hunk 1]
```diff
@@ -7,45 +6,17 @@
...
 
 ticklines = ax.get_xticklines()
-ticklines.extend( ax.get_yticklines() )
+ticklines.extend(ax.get_yticklines())
 gridlines = ax.get_xgridlines()
-gridlines.extend( ax.get_ygridlines() )
+gridlines.extend(ax.get_ygridlines())
 ticklabels = ax.get_xticklabels()
-ticklabels.extend( ax.get_yticklabels() )
+ticklabels.extend(ax.get_yticklabels())
```
Comment (WeatherGod) [inline examples/pylab_examples/axes_props.py]: Could we make these calls cleaner by just concatenating the x and y lists together? Something like `ticklines = ax.get_xticklines() + ax.get_yticklines()`?

Comment (ericmjl) [inline examples/pylab_examples/axes_props.py]: sure, I'll give that a shot.

**Comment by WeatherGod** [comment]
much better! Not exactly sure if it falls under MEP12, but many of these old examples haven't been updated in a long time to take advantage of newer APIs. Clear examples make for better documentation, so I am not against small modifications here and there to improve these scripts.

**Comment by ericmjl** [comment]
Okie dokes, no problem!

**Pull request merged**